searchcore/src/vespa/searchcore/proton/server/storeonlydocsubdb.cpp

```diff
@@ -426,9 +428,37 @@ StoreOnlyDocSubDB::reconfigure(const search::LogDocumentStore::Config & config,
     _rSummaryMgr->reconfigure(config);
 }
 
+namespace {
+
+constexpr double RETIRED_DEAD_RATIO = 0.5;
+
+struct UpdateConfig : public search::attribute::IAttributeFunctor {
+    void operator()(search::attribute::IAttributeVector &iAttributeVector) override {
+        auto attributeVector = dynamic_cast<search::AttributeVector *>(&iAttributeVector);
+        if (attributeVector != nullptr) {
+            auto cfg = attributeVector->getConfig();
+            cfg.setCompactionStrategy(search::CompactionStrategy(RETIRED_DEAD_RATIO, RETIRED_DEAD_RATIO));
```

Comment:
This has two issues:

1. normal config is not reapplied if node is moved from retired to up state.
2. new config without cluster state changes will set normal compaction strategy on the node even if it is still retiring.

Reply:
Yes, will address that.

Reply:
Now both issues should be addressed.

Reply:
The first issue is only addressed for document meta store and not for normal attribute vectors.

Reply:
Now it should address for both